frontend/src/Menu.js

```diff
@@ -21,13 +21,13 @@ function Menu({backend}) {
             />
           </Link>
         </Navbar.Brand>
-        <Authentication {...{backend}} />
+        <Authentication {...{ backend }} />
       </Container>
     </Navbar>
   );
 }
 
-function Authentication({backend}) {
+function Authentication({ backend }) {
```

Comment:
For your next ticket, please don't add lines or files unrelated with the ticket.